kpi/views/environment.py

```diff
@@ -43,38 +43,82 @@ class EnvironmentView(APIView):
         'COMMUNITY_URL',
         'FRONTEND_MIN_RETRY_TIME',
         'FRONTEND_MAX_RETRY_TIME',
-        ('FREE_TIER_DISPLAY', lambda value, request: json.loads(value)),
-        ('FREE_TIER_THRESHOLDS', lambda value, request: json.loads(value)),
-        ('PROJECT_METADATA_FIELDS', lambda value, request: json.loads(value)),
-        ('USER_METADATA_FIELDS', lambda value, request: json.loads(value)),
-        (
-            'SECTOR_CHOICES',
+    ]
+
+    @classmethod
+    def process_simple_configs(cls):
+        return {
+            key.lower(): getattr(constance.config, key)
+            for key in cls.SIMPLE_CONFIGS
+        }
+
+    JSON_CONFIGS = [
+        'FREE_TIER_DISPLAY',
+        'FREE_TIER_THRESHOLDS',
+        'PROJECT_METADATA_FIELDS',
+        'USER_METADATA_FIELDS',
+    ]
```

Comment:
I would have grouped "static" variables together at the top of the class and grouped the methods below that. 
Just my own preference.

Reply:
Thanks, but no thanks! These fields are grouped intentionally next to the method that operates on them. @noliveleger